adapters/appnexus/appnexustest/supplemental/invalid-bid-type.json

```diff
@@ -0,0 +1,121 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "app": {
+      "prebid": {
```

Comment:
Ughh! I missed this before but you need to move this to `app.ext.prebid` as well like you did in the other test.

Reply:
oh, I missed it. Updated it in [d9903b5](https://github.com/prebid/prebid-server/pull/2726/commits/d9903b50a4968f891769a5cecc3924366b660cb4)